hack/generator/pkg/codegen/testdata/ArmResource/Arm_test_dependent_resource_and_ownership.golden

```diff
@@ -18,6 +19,17 @@ type A struct {
 	Spec              A_Spec `json:"spec,omitempty"`
 }
 
+// +kubebuilder:webhook:path=/mutate-test-infra-azure-com-v20200101-a,mutating=true,sideEffects=None,matchPolicy=Exact,failurePolicy=fail,groups=test.infra.azure.com,resources=as,verbs=create;update,versions=v20200101,name=default.v20200101.as.test.infra.azure.com
+
+var _ admission.Defaulter = &A{}
+
+// Default defaults the Azure name of the resource to the Kubernetes name
+func (a *A) Default() {
```

Comment:
Not sure about the name `Default()` - would `ApplyDefaults()` be clearer?

Reply:
this is an interface method from `Defaulter` so cannot be renamed